presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoControlUtilsRpm.java

```diff
@@ -0,0 +1,222 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.teradata.prestomanager.agent;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.glassfish.jersey.client.JerseyClient;
+
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriBuilder;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.teradata.prestomanager.agent.AgentFileUtils.getFileProperty;
+import static com.teradata.prestomanager.agent.CommandExecutor.executeCommand;
+import static java.lang.String.format;
+import static javax.ws.rs.client.Entity.entity;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import static javax.ws.rs.core.Response.Status.ACCEPTED;
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.glassfish.jersey.client.JerseyClientBuilder.createClient;
+
+public final class PrestoControlUtilsRpm
+{
+    private static final Logger LOGGER = LogManager.getLogger(PrestoControlUtilsRpm.class);
+    private static final JerseyClient JERSEY_CLIENT = createClient();
+    private static final String LAUNCHER_SCRIPT = "/usr/lib/presto/bin/launcher";
+    private static final Path PRESTO_CONFIG_FILE = Paths.get("/etc/presto/config.properties");
+    private static final Gson GSON = new Gson();
+    private static final int SUBPROCESS_TIMEOUT = 120;
+
+    private PrestoControlUtilsRpm() {}
+
+    public static Response startUsingRpm()
+    {
+        try {
+            if (!isInstalled()) {
+                LOGGER.error("Presto is not installed");
+                return Response.status(NOT_FOUND).entity("Presto is not installed").build();
+            }
+        }
+        catch (PrestoManagerException e) {
+            LOGGER.error(e.getMessage(), e.getCause());
+            return Response.status(INTERNAL_SERVER_ERROR).build();
+        }
+        new Thread(() -> {
+            try {
+                int startPresto = executeCommand(SUBPROCESS_TIMEOUT, "service", "presto", "start");
+                if (startPresto != 0) {
+                    LOGGER.error("Failed to start presto");
+                }
+            }
+            catch (PrestoManagerException e) {
+                LOGGER.error(e.getMessage(), e.getCause());
+            }
+        }).start();
+        return Response.status(ACCEPTED).entity("Presto is being started. Please check back later using status.").build();
```

Comment:
Presto is being started. Please check back later using status to verify that Presto has successfully started.